src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.2.0")]
+[assembly: AssemblyFileVersion("2.2.0")]
+[assembly: AssemblyInformationalVersion("2.2.0")]

```

Comment:
Your "generate a VersionInfo.cs file" technique is another one I'm familiar with. We use it at the Day Job.

I'm indifferent to the mechanism we use for setting the version to tell the truth. I _do_ enjoy running `rake next_version[new_version]` to do all the half a dozen or so little steps we'd otherwise have to do as humans, but I don't really care how it accomplishes things under the hood (capot/bonnet). If you want a change, go for it